docs/user-guide/sync-options.md

```diff
@@ -142,7 +142,11 @@ spec:
 
 If the `Replace=true` sync option is set the Argo CD will use `kubectl replace` or `kubectl create` command to apply changes.
 
-This can also be configured at individual resource level.
+!!! danger
```

Comment:
@ashutosh16 Maybe we change this from Danger to Warning :). I think the outage comment is good and covers the severity of what could happen here.